react/features/base/lib-jitsi-meet/functions.js

```diff
@@ -52,11 +53,35 @@ export function isAnalyticsEnabled(stateful: Function | Object) {
             && Boolean(analyticsScriptUrls.length));
 }
 
+/**
+ * Determines whether a specific JitsiConferenceErrors instance indicates a
+ * fatal JitsiConference error.
+ *
+ * FIXME Figure out the category of errors defined by the function and describe
+ * that category. I've currently named the category fatal because it appears to
+ * be used in the cases of unrecoverable errors that necessitate a reload.
+ *
+ * @param {Object|string} error - The JitsiConferenceErrors instance to
+ * categorize/classify or an Error-like object.
+ * @returns {boolean} True if the specified JitsiConferenceErrors instance
+ * indicates a fatal JitsiConference error; otherwise, false.
+ */
+export function isFatalJitsiConferenceError(error: Object | string) {
+    if (typeof error !== 'string') {
+        error = error.name; // eslint-disable-line no-param-reassign
+    }
+
+    return (
+        error === JitsiConferenceErrors.FOCUS_LEFT
+            || error === JitsiConferenceErrors.FOCUS_DISCONNECTED
```

Comment:
`FOCUS_DISCONNECTED` should come before `FOCUS_LEFT` in order to keep adding to the list as easy as looking up a word in a dictionary (as our coding style states).